app/index.html

```diff
@@ -82,6 +82,7 @@
     <script src="bower_components/angular-bootstrap/ui-bootstrap-tpls.js"></script>
     <script src="bower_components/lodash/lodash.js"></script>
     <script src="bower_components/bootstrap/dist/js/bootstrap.js"></script>
+    <script src="bower_components/bootstrap-combobox/js/bootstrap-combobox.js"></script>
```

Comment:
I did a quick inventory, and I consider these suspicious PatternFly JS dependencies:

"bootstrap-combobox": "~1.1.6",
"bootstrap-datepicker": "~1.4.0",
"bootstrap-switch": "~3.3.2",
"bootstrap-touchspin": "~3.0.3",
"bootstrap-treeview": "~1.2.0",
"datatables-colreorder": "~1.1.3",
"datatables-colvis": "~1.1.2",
"moment": "~2.11.0"

That is, these are JS dependencies that are added from PatternFly that I don't think we make use of and have not explicitly been excluded in the wiredep exclude.

![screen shot 2016-07-06 at 9 33 13 am](https://cloud.githubusercontent.com/assets/895728/16619602/d09ef428-435c-11e6-818d-c09761e57c98.PNG)